Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/update/CCLConfigurationUpdater.kt

```diff
@@ -0,0 +1,61 @@
+package de.rki.coronawarnapp.ccl.configuration.update
+
+import androidx.annotation.VisibleForTesting
+import de.rki.coronawarnapp.covidcertificate.booster.BoosterRulesRepository
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toLocalDateUtc
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+import org.joda.time.Instant
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CCLConfigurationUpdater @Inject constructor(
+    private val cclSettings: CCLSettings,
+    private val boosterRulesRepository: BoosterRulesRepository,
+    // private val dccWalletManager: DccWalletInfoCalculationManager
```

Comment:
I think we should use the `DccWalletInfoUpdateTask`  here ?
/cc @chiljamgossow

Reply:
or actually the `DccWalletInfoUpdateTrigger` ?

Reply:
Do we need the task? otherwise calling the manager is fine

Reply:
As discussed, I am using the `DccWalletInfoUpdateTrigger` now.